test/common/testlib.py

```diff
@@ -634,8 +638,6 @@ def allow_restart_journal_messages(self):
 
     def allow_authorize_journal_messages(self):
         self.allow_journal_messages("cannot reauthorize identity.*:.*unix-user:admin.*",
-                                    "Error executing command as another user: Not authorized",
-                                    "This incident has been reported.",
```

Comment:
Oh, sorry, indentation -- I thought `allowed_messages` was set in `login_and_go()`, but it's a class member. So, all good!